e2e/tests/nightly/dataproc-to-gce.spec.ts

```diff
@@ -14,7 +14,7 @@ describe('Updating runtime compute type', () => {
   });
 
   test('Switch from dataproc to GCE', async () => {
-    await createWorkspace(page, { cdrVersion: config.ALTERNATIVE_CDR_VERSION_NAME });
+    await createWorkspace(page, { cdrVersionName: config.ALTERNATIVE_CDR_VERSION_NAME });
```

Comment:
Renamed parameter name `cdrVersion` to `cdrVersionName` in `test-utils` `createWorkspace` function.